### PR TITLE
TF-572: Update config via Camera Masking Screen

### PR DESCRIPTION
**Pull request by JordanBerni-UL**
- [x] Code Review
- [x] Developer Testing
- [x] QA Review
- [x] Test Plan updated
- [x] PO Review (Optional)
- [x] Update Release Notes

**Known Issues**

- Sliders only update on drag end
- Camera's sometimes flicker

Created https://ultrahaptics.atlassian.net/browse/TF-649 to look at BRS performance issues

### CODE REVIEW

File: TF_Settings_Web/src/Components/Controls/ToggleSwitch.tsx [hunk 1]
```diff
@@ -4,32 +4,26 @@ import React from 'react';
...
-        <input
-            type="checkbox"
-            style={{ display: 'none' }}
-            checked={value}
-            onChange={() => {
-                onChange(!value);
-            }}
-        />
+        {/* eslint-disable-next-line @typescript-eslint/no-empty-function */}
+        <input type="checkbox" style={{ display: 'none' }} checked={value} onChange={() => {}} />
```
Comment (NatGreenwood-UL) [inline TF_Settings_Web/src/Components/Controls/ToggleSwitch.tsx:14]: Can these ToggleSwitches no longer be changed? Seems like they wouldn't do anything on click now?

File: TF_Settings_Web/src/TouchFree/Configuration/ConfigurationManager.ts [hunk 2]
```diff
@@ -62,14 +62,14 @@ export class ConfigurationManager {
     public static RequestConfigFileChange(
         _interaction: Partial<InteractionConfig> | null,
```
Comment (NatGreenwood-UL) [inline TF_Settings_Web/src/TouchFree/Configuration/ConfigurationManager.ts:63]: All the changes to the TouchFree Tooling applied from here down in the listed changes need to be replicated across to the "main" version of the TouchFree tooling before we end up with really divergent versions.

Comment (NatGreenwood-UL) [reply TF_Settings_Web/src/TouchFree/Configuration/ConfigurationManager.ts:63]: Copying one atop the other they seem to have quite a few differences: 
image

File: TF_Settings_Web/src/Components/Controls/ToggleSwitch.tsx (hunk 1 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/Controls/ToggleSwitch.tsx:14]: The label around the ToggleSwitch takes care of what to do when the checkbox is clicked (TF doesn't trigger the onChange function and that would only apply to the actual switch element, not the surrounding label).

I will add this as a comment to make it more clear

**Comment by cartmouse** [comment]
I've had a look over these latest changes and it all looks good to me code wise!

**Review by cartmouse — APPROVED** [review]
I've had a look over these latest changes and it all looks good to me code wise!

**Comment by cartmouse** [comment]
Reviewed up to here, all looking good!

**Pull request merged**